core/new-gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -731,10 +733,16 @@ export class WorkflowEditorComponent implements AfterViewInit {
 
     // if port is already connected, do not allow another connection, each port should only contain at most 1 link
```

Comment:
better name for this function would be isConnectionValid?

Reply:
I think current `validateOperatorConnection` is also fine, because this name comes fromt JointJS I believe, so we are maintaining the consistency with the library.